webapp-django/crashstats/crashstats/views.py

```diff
@@ -1425,6 +1425,18 @@ def handle_middleware_404(crash_id, error_code):
         ]
     context['raw_keys'].sort(key=unicode.lower)
 
+    context['install_time'] = None
```

Comment:
Actually, this should just be ``context['install_time'] = context['raw'].get('InstallTime')``. The conversion to a date is done in the template. 

I feel terrible for merging something that obviously broken in master. Thanks for catching it!